test/postgrator-cli-tests.js

```diff
@@ -237,8 +243,7 @@ var buildTestsForOptions = function (options) {
         postgratorCli.run(options, function(err, migrations) {
             restoreOptions();
             assert.ifError(err);
-            assert.equal(migrations.length, 4);
-            assert.equal(migrations[migrations.length-1].version, 4);
+            assert.equal(migrations.length, 0);
```

Comment:
Doesn't this change test assertion? Shouldn't it migrate 4 files here like before?

Reply:
In version 3 of postgrator it only put the migrate object in the array when effectively execute the migration, if it only check the hash nothing is generated